lms/djangoapps/dashboard/sysadmin.py

```diff
@@ -43,7 +45,7 @@
 from xmodule.modulestore.xml import XMLModuleStore
 from opaque_keys.edx.locations import SlashSeparatedCourseKey
 from instructor_task.models import InstructorTask
-
```

Comment:
Should we restore this newline line between the import and the `log = logging.getLogger(__name__)` call?